sphinx/api/configuration/suite.rst

```diff
@@ -136,16 +136,17 @@ A suite directory may contain the following:
 
       .. deprecated:: 2015.04
 
-         Equivalent to :rose:conf`root-dir{work}=LIST`.
+         Equivalent to :rose:conf:`root-dir{work}=LIST`.
```

Comment:
Damn, I wish sphinx would pick these up automatically. Thanks for fixing this.

Reply:
It would certainly be a useful functionality for sphinx. I did do a grep for `:rose:conf` + a backtick so there shouldn't be any more!

Reply:
Great, thanks.